doc/ch-test_desc.rst

```diff
@@ -47,8 +51,8 @@ tested on different systems by copying the necessary artifacts between them,
 e.g. by building images on one system and running them on another. The *scope*
 allows trading off thoroughness versus time.
 
-:code:`PHASE` must be one of the following; the default is to run
-:code:`build`, :code:`run`, and :code:`examples` in that order.
+:code:`PHASE` must be one of the following; :code:`build`, :code:`run`,
+:code:`examples`, or :code:`all`.
```

Comment:
`PHASE` can be anything in the bullet list; the clause after the semicolon can be removed.